platform-hub-api/spec/services/agents/git_hub_agent_service_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+describe Agents::GitHubAgentService, type: :service do
+
+  let :token do
+    'foo_token'
+  end
```

Comment:
Hm, weird… my Ruby linting tells me your example is invalid (!) – this may be why I got into the habit of just using the `do … end` style of blocks. (it should be using plain old rubocop under the hood, so not sure why it's complaining).